module-2/lab-matplotlib-seaborn/your-code/Readme.md

```diff
@@ -0,0 +1,39 @@
+# Matplot y Seaborn
+
+Estas son las cuestiones resultantes del lab:
+
+### Challenge 1
+
+Respuestas a 3, 5 y 6 por ej. salen con un código como este: 
+
+```
+[<matplotlib.lines.Line2D at 0x7f010b0d0da0>]
+```
+
+Esto no parece una descripción de objeto, parece sólo de la posición en memoria. Porqué sale?  (Entiendo que si hago una asignación dejará de salir, pero en qué casos sale? 
+
+Indexex below some binned histograms, don't  correspond to the graphic. For example the distribution of 'Age' variable with 10, 20, 50 bins, see the biggest bins clearly don fit the numbers. 
+
+
+
+#### Use the appropriate plot to visualize the column `Gender`
+
+How do I set different colors for each bar. Seems I have to use a palette? 
+
+#### Use the appropriate plot to visualize the column `Pclass`
```

Comment:
If you do the following way, matplotlib set one different color for each bar/Pclass. If you want to change the color you can do it using the property color= inside the plot function. The way it works is described quite well in matplotlib library, here it is the link (https://matplotlib.org/api/_as_gen/matplotlib.pyplot.plot.html)
```Python
titanic['Pclass'].value_counts().plot(kind='bar')
plt.xlabel('Class')
plt.ylabel('Count')
```